packages/perseus/src/widgets/interactive-graphs/graphs/components/use-control-point.tsx

```diff
@@ -92,10 +93,20 @@ export function useControlPoint(params: Params): Return {
             y: srFormatNumber(point[Y], locale),
         });
 
+    // Set the forwarded ref to the focusable handle element when it changes.
     useLayoutEffect(() => {
         setForwardedRef(forwardedRef, focusableHandleRef.current);
     }, [forwardedRef]);
 
+    // If the point is being dragged and is not focused, focus the focusable handle.
+    useLayoutEffect(() => {
+        if (dragging && !focused) {
+            // If the point is being dragged, focus the focusable handle so that
+            // users can continue to interact with the point using the keyboard or buttons.
+            focusableHandleRef.current?.focus();
+        }
```

Comment:
Question: Should there also be a `setFocused(true)` here to update the stored state after setting the focus?

Reply:
I thought so too, but I tested it out and by manually triggering the focus, we also trigger the onFocus event on that element that calls the setFocused value already!